src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -174,26 +187,52 @@ public function setConfig(GeneratorConfigInterface $config)
      */
     public function getConfig()
     {
-        if ($this->config === null) {
+        if (!isset($this->config)) {
```

Comment:
What's the reason behind this change? `!isset` is slower than `=== null`

Reply:
idem

Reply:
Fixed.